### PR TITLE
lib/mergeset: adds tracking for indexdb records drop

### PR DESCRIPTION
**Pull request by f41gh7**
It allows to create alert for possible item drops at indexdb. It may happen, if ingested metric size exceeds max indexdb item size.

### CODE REVIEW

File: app/vmstorage/main.go [hunk 1]
```diff
@@ -492,6 +493,7 @@ func writeStorageMetrics(w io.Writer, strg *storage.Storage) {
 
 	metrics.WriteCounterUint64(w, `vm_indexdb_items_added_total`, idbm.ItemsAdded)
 	metrics.WriteCounterUint64(w, `vm_indexdb_items_added_size_bytes_total`, idbm.ItemsAddedSizeBytes)
+	metrics.WriteCounterUint64(w, `vm_indexdb_items_dropped_total{reason="too_long_item"}`, m.IndexDBMetrics.TooLongItemsDroppedTotal)
```
Comment (zekker6) [inline app/vmstorage/main.go:496]: nit:
```suggestion
	metrics.WriteCounterUint64(w, `vm_indexdb_items_dropped_total{reason="too_long_item"}`, idbm.TooLongItemsDroppedTotal)
```

**Review by zekker6 — APPROVED** [review]
LGTM

**Review by hagen1778 — CHANGES_REQUESTED** [review]
@f41gh7 would you mind adding an alerting rule as well?

**Comment by hagen1778** [comment]
@f41gh7 please see my follow-up here https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6297/commits/c78adacde10458afdb86001a99e93a76828355ea

**Review by hagen1778 — APPROVED** [review]

**Comment by valyala** [comment]
@f41gh7 , see the follow-up commit 832e0886595775784053cd89f2544350464d13b9

**Pull request merged**